SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialSender.h

```diff
@@ -118,6 +122,7 @@ class SPATIALGDK_API USpatialSender : public UObject
 	FOutgoingRepUpdates HandoverObjectToUnresolved;
 
 	FOutgoingRPCMap OutgoingRPCs;
```

Comment:
Like I said in the comment above, investigate if this is still needed.

Reply:
As @improbable-valentyn mentioned that it might be necessary for singletons in multi-worker environment - I'm keeping it for now.